File: koserver-common/src/koserver/common/enums/LoginResult.java
package koserver.common.enums;

public class LoginResult {

    public static final int AUTH_SUCCESS = 0x01;
    public static final int AUTH_NOT_FOUND = 0x02;
    public static final int AUTH_INVALID = 0x03;
    public static final int AUTH_BANNED = 0x04;
    public static final int AUTH_IN_GAME = 0x05;
    public static final int AUTH_ERROR = 0x06;
    public static final int AUTH_AGREEMENT = 0xF;
    public static final int AUTH_FAILED = 0xF;
}

File: KnightOnline.Domain/Enums/LoginResult.cs
namespace KnightOnline.Domain.Enums
{
    public enum LoginResult : byte // Or int if values might exceed 255
    {
        AuthSuccess = 0x01,
        AuthNotFound = 0x02,
        AuthInvalid = 0x03,
        AuthBanned = 0x04,
        AuthInGame = 0x05,
        AuthError = 0x06,
        AuthAgreement = 0x0F,
        AuthFailed = 0x0F // Same value as AuthAgreement
    }
}
